packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -250,13 +251,49 @@ export default class QuanticResultQuickview extends LightningElement {
   }
 
   setFocusToTop() {
+    /** @type {HTMLElement} */
     const focusTarget = this.template.querySelector(
       `.slds-button.slds-button_icon`
     );
 
     if (focusTarget) {
-      // @ts-ignore
       focusTarget.focus();
     }
   }
+
+  /**
+   * @param {KeyboardEvent} evt
+   */
+  onCloseKeyDown(evt) {
+    if (evt.shiftKey && evt.code === 'Tab') {
+      evt.preventDefault();
+      const lastFocusableElement =
+        this.lastFocusableElementInFooterSlot ||
+        this.lastFocusableElementInQuickview;
+      if (lastFocusableElement) {
+        lastFocusableElement.focus();
+      } else {
+        this.setFocusToHeader();
+      }
+    }
+  }
+
+  get lastFocusableElementInFooterSlot() {
+    /** @type {HTMLSlotElement} */
+    const footerSlot = this.template.querySelector('slot[name=footer]');
+    if (footerSlot) {
+        const lastElement = getLastFocusableElement(footerSlot);
+        if (lastElement) return lastElement;
```

Comment:
won't `lastElement` be null anyway if there is none?
Just `return lastElement;`